cabal-testsuite/PackageTests/NewConfigure/cabal.project.local

```diff
@@ -0,0 +1,26 @@
+verbose: verbose +nowrap +markoutput
```

Comment:
Yeah, I don't think we care about what new-configure actually outputs (which, looking here, is quite a lot of things!) We just care that the old contents of cabal.project.local were preserved.